source/server/spongineer/properties.rst

```diff
@@ -0,0 +1,74 @@
+|                                    | where each transaction is created, potentially adding more information if there |
+| (boolean, default: false)          | is an exception thrown during the processing of said transactions. Should       |
+|                                    | always be included for logs when reporting issues involving these sort of       |
+|                                    | transactions.                                                                   |
++------------------------------------+---------------------------------------------------------------------------------+
+| **sponge.filter.debug**            | Writes out classes that are generated by the filter generator to the            |
+|                                    | ``.sponge.debug.out`` folder, named after the class they are altering.          |
+| (boolean, default: false)          |                                                                                 |
++------------------------------------+---------------------------------------------------------------------------------+
+| **sponge.print_all_catalog_types** | Prints out all registered ``CatalogType`` s and their ``String`` id's at the    | 
+|                                    | end of startup, useful for finding out what id's are registered from vanilla,   |
+| (boolean, default: false)          | mods, and plugins for all created ``CatalogType`` s.                            |
++------------------------------------+---------------------------------------------------------------------------------+
+|                                    | Prints out a very spam filled log for every block captured that is processed,   |
+|                                    | information about what ``BlockStates`` are proxied, what ``BlockStates`` exist  |
+| **sponge.debugBlockTransactions**  | on the ``Chunk`` during processing, and potentially what ``BlockStates``        |
+|                                    | existed prior to a future transaction changing positions. Likewise applies to   |
+| (boolean, default: false)          | tracking ``TileEntity`` instances. Useful for debugging purposes for the        |
+|                                    | implementation, and potentially plugins, when dealing with                      |
+|                                    | ``ChangeBlockEvents`` and their processing.                                     |
++------------------------------------+---------------------------------------------------------------------------------+
+
+.. warning::
+   **sponge.debugcauseframes** has the potential to increase performance costs due to the stack trace creation for
+   every frame in the duration of the game's lifecycle.
+
+   **sponge.debugProxyChanges** has the same potential to increase performance costs in potentially intensive 
+   environments due to the exception creation for every potential ``Entity`` / ``TileEntity`` / ``Block`` being ticked 
```

Comment:
If you remove the `the` (before `exception`) it becomes clearer, imho.

Reply:
Removed `the` .